driver/pelican/main.go

```diff
@@ -244,6 +255,22 @@ func main() {
 			}
 		}()
 
+		go func() {
+			for {
+				if schedStatus, schedErr := currentPelican.GetSchedule(sitename); schedErr != nil {
+					fmt.Printf("Failed to retrieve Pelican's Schedule: %v\n", schedErr)
+				} else if schedStatus != nil {
```

Comment:
Is it ever the case that `GetSchedule` returns a `nil` schedule when there is no error?

Reply:
Looking at the code, it seems that as long as there's no error, there will be a schedule status, so I removed the conditional. Thanks for the catch!